### ISSUE TITLE
Hi I am running a code for text classification in colab.versions for different packages are ktrain- 0.33.3 tensorflow 2.11.1 eli5 0.13.0 ipythonn 7.9.0 i am not getting highlighted text as the output when i use predictor.explain()

### ISSUE THREAD

**Issue by PriyankaVivek** [amaiya/ktrain#483]

**Comment by amaiya** [comment]
It looks like there is some issue with newer versions of `scikit-learn` and the `eli5` library. 

The temporary workaround is to downgrade `scikit-learn` to a version than less than or equal to v1.0.2:

```
pip install sckit_learn==1.0.2
```

**Comment by PriyankaVivek** [comment]
@amaiya Thanks a lot .it is working now

**Comment by amaiya** [comment]
If you uninstall `eli5` and re-install the `eli5-tf` fork, things should highlight correctly using `scikit-learn==1.2.2`:
```
pip install https://github.com/amaiya/eli5-tf/archive/refs/heads/master.zip
```
